src/app/pages/profile/profile.module.ts

```diff
@@ -24,9 +27,10 @@ import { RoleTransformPipe } from 'src/app/shared/pipes/role.pipe';
     ReactiveFormsModule,
     HttpClientModule,
     NgxSpinnerModule,
-		AuthLayoutModule,
+    AuthLayoutModule,
+    RoundProgressModule
   ],
-  providers: [],
+  providers: [ProfileService],
```

Comment:
@shemaeric The ProfileService is already provided in the **root** module via its **Injectable** decorator. Declaring it here again will make us lose the advantage in **tree-shaking** and we may not have the **singleton** we expect it to be.

Reply:
Thank you @Victoradukwu I have worked on it.